nix/kubernetes/cluster.nix

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
This file's more verbose than I was expecting, that because of the key wrangling?

Reply:
Yeah, this file like `certs.nix` will mostly be brought into the main nixpkgs repo eventually.